engines/stark/ui/gamewindow.cpp

```diff
@@ -68,6 +76,11 @@ void GameWindow::onRender() {
 		// Go for the next one
 		element++;
 	}
+
+	float fadeLevel = StarkScene->getFadeLevel();
+	if (fadeLevel < 1.0) {
+		_fadeRenderer->render(fadeLevel);
```

Comment:
Looks like some floating point error prevents this check from working as intended when fading is complete:
![image](https://cloud.githubusercontent.com/assets/52294/24826737/fd12ba44-1c3c-11e7-8a8a-a225cb785c62.png)